src/zippo/core.cljc

```diff
@@ -149,7 +149,8 @@
     ;; MapEntry doesn't support meta
     (map-entry? node)
     (let [[k v] children]
-      (new clojure.lang.MapEntry k v))
```

Comment:
This one is tricky as it mixes with a regular vector. Let me think for a while...